azure-pipelines.yml

```diff
@@ -1,28 +1,77 @@
-# Android
-# Build your Android project with Gradle.
-# Add steps that test, sign, and distribute the APK, save build artifacts, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/languages/android
-
+variables:
+  sdk: 'iphoneos'
+  configuration: 'Build'
+  msMapsModulesPath: 'iOS/MSMapsModules'
+  msMapsModulesToRootPath: '../../'
+  msMapsModulesWorkspacePath: '$(msMapsModulesPath)/MSMapsModules.xcworkspace'
+  testAppKeysPath: '$(msMapsModulesPath)/MSMapsModulesTestApp/keys.plist'
+  referenceAppPath: 'iOS/MSMapsModulesReferenceApp'
+  referenceAppWorkspacePath: '$(referenceAppPath)/MSMapsModulesReferenceApp.xcworkspace'
+  
 trigger:
 - master
 
-pool:
-  vmImage: 'windows-latest'
-
-steps:
-
-  # Create dummy secrets.gradle
-- bash: |
-    echo ext.credentialsKey = "\"DummyKey\"" > secrets.gradle
-    echo `<secrets.gradle`
-  displayName: Create secrets.gradle
-  workingDirectory: 'Android/Samples/app'
+jobs:
+- job: Build_Android
+  pool:
+    vmImage: 'windows-latest'
+  steps:
+    # Create dummy secrets.gradle
+  - bash: |
+      echo ext.credentialsKey = "\"DummyKey\"" > secrets.gradle
+      echo `<secrets.gradle`
+    displayName: Create secrets.gradle
+    workingDirectory: 'Android/Samples/app'
+  - task: Gradle@2
+    inputs:
+      workingDirectory: 'Android/Samples'
+      gradleWrapperFile: 'Android/Samples/gradlew'
+      gradleOptions: '-Xmx3072m'
+      publishJUnitResults: false
+      testResultsFiles: '**/TEST-*.xml'
+      tasks: 'build'
 
-- task: Gradle@2
-  inputs:
-    workingDirectory: 'Android/Samples'
-    gradleWrapperFile: 'Android/Samples/gradlew'
-    gradleOptions: '-Xmx3072m'
-    publishJUnitResults: false
-    testResultsFiles: '**/TEST-*.xml'
-    tasks: 'build'
+- job: Build_iOS
+  pool:
+    vmImage: 'macOS-10.14'
+  steps:
+  - bash: |
```

Comment:
You can add format step and fail the build if the sources are not properly formatted. Below is what we do for Beacon. Adjust as necessary.

# Check source code formatting
- bash: |
    brew install clang-format
    cd src/iOS/
    ./format-all
    ../../BuildScripts/CheckForChanges.sh "iOS code is not properly formatted; run src/iOS/format-all"
  displayName: Check source code formatting